client/src/components/Metrics/MetricsEpoch/MetricsEpochGridFundsUsage/MetricsEpochGridFundsUsage.tsx

```diff
@@ -33,7 +33,9 @@ const MetricsEpochGridFundsUsage: FC<MetricsEpochGridFundsUsageProps> = ({
 
   const projectCosts = epochInfo ? epochInfo.operationalCost : BigInt(0);
   const staking = epochInfo ? epochInfo.staking : BigInt(0);
-  const ppf = epochInfo ? epochInfo.ppf : BigInt(0);
+  // const ppf = epochInfo ? epochInfo.ppf : BigInt(0);
+  // Temporary ppf forced by backend changes. Correct formula above!
```

Comment:
Please ticket a ticket and place a TODO for removal of this workaround.

Reply:
@aziolek Done